packages/styled-components/components/searchForm/index.js

```diff
@@ -51,7 +52,13 @@ const SearchForm = (props) => {
       action={baseUrl}
       onSubmit={handleSubmit(onSubmit)}
     >
+      <SearchLabel
+        id="search-label"
+      >
+        Search input
+      </SearchLabel>
```

Comment:
I noticed in the JFC repo that eslint wanted me to wrap the label around the input. This error would not go away until I did: `A form label must be associated with a control  jsx-a11y/label-has-associated-control`. 

Just a note, maybe for a future conversation with how we set up that rule. I don't think that should be forced, but I did it anyway there to make eslint shut up.

Reply:
gotta keep eslint happy! made that change